### PR TITLE
Use futures-await

### PR DESCRIPTION
**Pull request by rossnomann**

### CODE REVIEW

**Comment by rossnomann** [comment]
@savant2212 so you should unapprove PR and wait until I will update it.

File: src/main.rs [hunk 1]
```diff
@@ -32,33 +37,42 @@ const ESSENCE_OUT: &str = "
...
+    let test_user = UserId::new(560120889); // test test
+    let essence_disabled = vec![evengining_user, test_user];
+
+    #[async]
+    for update in api.stream() {
         if let UpdateKind::Message(message) = update.kind {
             if message.from.id == test_user {
-                api.spawn(message.text_reply(POSHEL_NAHUY));
+                if let Ok(reply_msg) = await!(api.send(message.text_reply(POSHEL_NAHUY))) {
+                    let _ = await!(Delay::new(Duration::from_secs(30)));
```
Comment (mersinvald) [inline src/main.rs]: Is that necessary? O_o

Comment (mersinvald) [reply src/main.rs]: let _ I mean

File: src/main.rs [hunk 2]
```diff
@@ -32,33 +37,42 @@ const ESSENCE_OUT: &str = "
...
+                }
             } else {
-                if let MessageKind::Text { ref data, .. } = message.kind {
-                    let data = data.to_lowercase();
-                    if data.contains(ESSENCE_IN) {
-                        api.spawn(message.text_reply(
+                let msg = message.clone();
+                if let MessageKind::Text { data, .. } = message.kind {
+                    if data.to_lowercase().contains(ESSENCE_IN) {
+                        let _ = await!(api.send(msg.text_reply(
```
Comment (mersinvald) [inline src/main.rs]: Same here

File: src/main.rs (hunk 1 above)
Comment (rossnomann) [reply src/main.rs]: Compiler warns about unused result otherwise

**Pull request merged**